dockerfiles/che-cli/files-to-install/che.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/sh
```

Comment:
It's for the CLI.  `che.bat` and `che.sh` are files that you can install on linux / windows that will simplify the CLI down to `che start` so that you do not have to use docker syntax.  The `che-clie` image installs these files onto your computer. We'll also probably add in some sort of sudo yum install command for these files as well.